src/plugins/inspector/public/plugin.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import * as React from 'react';
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from '../../../core/public';
+import { InspectorViewRegistry } from './view_registry';
+import { Adapters, InspectorOptions, InspectorSession } from './types';
+import { InspectorPanel } from './ui/inspector_panel';
+
+export interface Setup {
+  registerView: InspectorViewRegistry['register'];
+
+  __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
```

Comment:
This is only here for shimming in legacy land, right?  We can remove once everyone imports via NP?  The name is hilarious, but it might actually be more descriptions if it was something like `__LEGACY__`, (if that really is the case).  😆

Reply:
Yes, we can remove this once no one uses `viewRegistry` anymore, but uses `registerView()` instead.

I actually like this name, its what React uses and it actually feels more descriptive than just "legacy". And its also fun.

We are already using the same name in the NP `data` plugin Expressions service. I would prefer keeping it, if you are OK with that.

Reply:
so funny, I didn't know React used it.  yea, that sounds fine to me, I don't feel strongly about the name.  And it is funny. 😄